swank/src/test/scala/org/ensime/server/protocol/swank/SwankFormatsSpec.scala

```diff
@@ -368,7 +367,7 @@ class SwankFormatsSpec extends FlatSpec with Matchers with EnsimeTestData {
     )
     marshal(
       DebugExceptionEvent(33L, dtid, "threadNameStr", Some(sourcePos1.file), Some(sourcePos1.line)): EnsimeEvent,
-      s"""(:debug-event (:type exception :exception 33 :thread-id "13" :thread-name "threadNameStr" :file $file1_str :line 57))"""
+      s"""(:debug-event (:type exception :exception 33 :thread-id "13" :thread-name "threadNameStr" :file "$file1" :line 57))"""
```

Comment:
The "marshal" tests fail: marshalling escapes backslashes which means that the expected string isn't equal to "$file1"